packages/compiler/test/test-parser-utils.ts

```diff
@@ -16,6 +12,7 @@ describe("compiler: parser utils", () => {
     ): Promise<{ root: CadlScriptNode; node: Node | undefined }> {
       const { source, pos } = extractCursor(sourceWithCursor);
       const root = parse(source);
+      dumpAST(root);
```

Comment:
for debug?

Reply:
I think that's something we could add to eslint ban-types rule as a warning like console.log so we don't leave it

Reply:
I think that's something we could add to eslint ban-types rule as a warning like console.log so we don't leave it

Reply:
This is actually ok to leave because dumpAST uses logVerboseTestOutput. So you can look at the output in test explorer but it doesn't pollute the console unless you set an env var. The function name could maybe be improved, though.

Reply:
i see